src/main/scala/com/raphtory/core/storage/pojograph/PojoGraphLens.scala

```diff
@@ -45,6 +46,7 @@ final case class PojoGraphLens(
 
   def setFullGraphSize(size: Int): Unit = {
     fullGraphSize = size
+    StorageTelemetry.pojoLensGraphSize.set(size)
```

Comment:
I believe we set the full Graph size in the query components no? It may be worth setting here the local size of the graph to give an idea of partition distribution per job